docs/content/preview/releases/techadvisories/ta-20648.md

```diff
@@ -30,9 +30,14 @@ The impacted releases contain an issue where an index update can be wrongly appl
 
 If you're using a {{<release "2.20">}} series release prior to {{<release "2.20.1.3">}}, it is recommended that you upgrade to YugabyteDB {{<release "2.20.1.3">}} or later. The fix ensures new writes to get executed correctly. But it will not repair existing inconsistent data. To fix inconsistent indexes, the indexes need to be recreated. It is advisable to create a new index, and then drop the older one.
 
-<!-- TODO: @prem
 ## Details
```

Comment:
@aishwarya24  - Can we remove this section. I've rework this ..  Apart from this, we can merge it . LGTM